images/build/Dockerfiles/Dockerfile

```diff
@@ -184,15 +184,22 @@ ENV PYTHONIOENCODING UTF-8
 RUN cd ${IMAGES_DIR} \
     && . ${BUILD_DIR}/__pythonVersions.sh \
     && ./installPlatform.sh python $PYTHON27_VERSION \
-    && ./installPlatform.sh python $PYTHON36_VERSION
+    && ./installPlatform.sh python $PYTHON36_VERSION \
+    && ./installPlatform.sh python $PYTHON38_VERSION \
+    && ./installPlatform.sh python $PYTHON39_VERSION
 RUN . ${BUILD_DIR}/__pythonVersions.sh && set -ex \
  && [ -d "/opt/python/$PYTHON27_VERSION" ] && echo /opt/python/$PYTHON27_VERSION/lib >> /etc/ld.so.conf.d/python.conf \
  && [ -d "/opt/python/$PYTHON36_VERSION" ] && echo /opt/python/$PYTHON36_VERSION/lib >> /etc/ld.so.conf.d/python.conf \
+ && [ -d "/opt/python/$PYTHON27_VERSION" ] && echo /opt/python/$PYTHON38_VERSION/lib >> /etc/ld.so.conf.d/python.conf \
+ && [ -d "/opt/python/$PYTHON36_VERSION" ] && echo /opt/python/$PYTHON39_VERSION/lib >> /etc/ld.so.conf.d/python.conf \
```

Comment:
I think here versions need to be matched

Reply:
good catch